_sass/blog.scss

```diff
@@ -112,6 +112,13 @@
   padding: 1rem;
   margin-bottom: 1rem;
 }
+.previous-btn{
+position: absolute;
+margin-left:-7em;
```

Comment:
this doesn't look good giving negative value and in em. Please describe why? 
Add a screenshot